### PR TITLE
Upgrade @azure/abort-controller version to ^2.1.2

### PR DESCRIPTION
**Pull request by jeremymeng**

### CODE REVIEW

File: packages/autorest.typescript/package.json [hunk 1]
```diff
@@ -80,7 +80,7 @@
   },
   "devDependencies": {
     "@azure-tools/test-recorder": "^3.0.0",
-    "@azure/abort-controller": "^1.0.1",
+    "@azure/abort-controller": "^2.1.2",
```
Comment (MaryGao) [inline packages/autorest.typescript/package.json:83]: Could you update other places also?
- https://github.com/Azure/autorest.typescript/blob/main/packages/rlc-common/src/metadata/packageJson/azurePackageCommon.ts#L45-L50
- https://github.com/Azure/autorest.typescript/blob/main/packages/typespec-ts/package.json#L63

Comment (jeremymeng) [reply packages/autorest.typescript/package.json:83]: Will do!

Comment (jeremymeng) [reply packages/autorest.typescript/package.json:83]: Done

**Review by joheredi — APPROVED** [review]

**Review by MaryGao — APPROVED** [review]
LGTM

**Pull request merged**